docs/src/tutorial.md

```diff
@@ -0,0 +1,109 @@
+# Tutorial
+
+## Installation
+
+[**MeshIntegrals.jl**](https://github.com/JuliaGeometry/MeshIntegrals.jl) is
+registered in the official Julia General registry, enabling it to be easily
+installed using Julia on an internet-connected computer.
+**MeshIntegrals.jl** should usually be installed in conjunction with
+[**Meshes.jl**](https://github.com/JuliaGeometry/Meshes.jl) and
+[**Unitful.jl**](https://github.com/PainterQubits/Unitful.jl) which provide
+support for geometries and physical units, respectively.
+
+**MeshIntegrals.jl** can be installed while using Julia by entering `pkg` mode
+with the `]` key and `add`ing it by name
+```julia-repl
+pkg> add MeshIntegrals
+```
+
+or by calling the built-in **Pkg.jl** library.
+```julia
+import Pkg
+Pkg.add("MeshIntegrals")
+```
+
+## Basic Usage
+
+Usage of **MeshIntegrals** typically also involves using **Meshes** and **Unitful**,
```

Comment:
```suggestion
Usage of **MeshIntegrals.jl** typically also involves using **Meshes.jl** and **Unitful.jl**,
```
I'd argue the `.jl` is part of the name, so it should always be attached when referring to the package's name.